src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -78,16 +78,8 @@ public bool IgnoreSheetNamesAtImport
 
       public IReadOnlyList<DataRepository> ImportExcelFromConfiguration(
          string configurationPath,
-         string dataPath)
-      {
-         return _dataImporter.ImportFromConfiguration(
-            GetConfiguration(configurationPath),
-            _metaDataCategories,
-            _columnInfos,
-            _dataImporterSettings,
-            dataPath
-         ).ToArray();
-      }
+         string dataPath) =>
+         ImportExcelFromConfiguration(GetConfiguration(configurationPath), dataPath);
```

Comment:
I am calling one overload from the other. It's typical to only have one implementation and the other are just calling the real implementation with other parameters (This is just a refactoring, nothing to do with the fix)